<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SniffSettings</class>
 <widget class="QWidget" name="SniffSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxSniffSettings">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
border: none;
}

QGroupBox::title {
    subcontrol-origin: margin;
}
QGroupBox::indicator:unchecked {
 image: url(:/icons/icons/collapse.svg)
}
QGroupBox::indicator:checked {
 image: url(:/icons/icons/uncollapse.svg)
}</string>
     </property>
     <property name="title">
      <string>Sniff settings</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_sniff_Signal">
           <property name="text">
            <string>Use values from:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBox_sniff_signal">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_sniff_use_signal">
             <property name="text">
              <string>Use</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_sniff_Noise">
           <property name="text">
            <string>Noise:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="spinbox_sniff_Noise">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxAdaptiveNoise">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With adaptive noise URH will update the noise level automatically during RX. This is helpful in a dynamic environment where noise differs in time.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Adaptive</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_sniff_Center">
           <property name="text">
            <string>Center:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spinbox_sniff_Center">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-3.140000000000000</double>
           </property>
           <property name="maximum">
            <double>3.140000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_sniff_BitLength">
           <property name="text">
            <string>Bit Length:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinbox_sniff_BitLen">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_sniff_Tolerance">
           <property name="text">
            <string>Error Tolerance:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spinbox_sniff_ErrorTolerance">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_sniff_Modulation">
           <property name="text">
            <string>Modulation:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="combox_sniff_Modulation">
           <item>
            <property name="text">
             <string>ASK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FSK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PSK</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_sniff_encoding">
           <property name="text">
            <string>Encoding:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox_sniff_encoding">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_sniff_viewtype">
           <property name="text">
            <string>View:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QComboBox" name="comboBox_sniff_viewtype">
             <item>
              <property name="text">
               <string>Bit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ASCII</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sniff_Timestamp">
             <property name="text">
              <string>Show Timestamp</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_sniff_OutputFile">
           <property name="text">
            <string>Write bitstream to file:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_sniff_OutputFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>None</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>groupBoxSniffSettings</tabstop>
  <tabstop>spinbox_sniff_Noise</tabstop>
  <tabstop>spinbox_sniff_Center</tabstop>
  <tabstop>spinbox_sniff_BitLen</tabstop>
  <tabstop>spinbox_sniff_ErrorTolerance</tabstop>
  <tabstop>combox_sniff_Modulation</tabstop>
  <tabstop>comboBox_sniff_encoding</tabstop>
  <tabstop>comboBox_sniff_viewtype</tabstop>
  <tabstop>checkBox_sniff_Timestamp</tabstop>
  <tabstop>lineEdit_sniff_OutputFile</tabstop>
 </tabstops>
 <resources>
  <include location="urh.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>groupBoxSniffSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
